src/abstract-interpretation/data-frame/absint-info.ts

```diff
@@ -16,6 +16,10 @@ interface DataFrameInfoBase {
 	domain?: DataFrameStateDomain
 }
 
+export interface DataFrameUnassignedInfo {
```

Comment:
We should add a comment to clarify whether this refers to completely unbound data frames, anonymous data frames, etc.